backend/tests/v5/unit/processors/teamspaces/projects/models/containers.test.js

```diff
@@ -344,15 +344,26 @@ const testDeleteContainer = () => {
 
 const testGetRevisions = () => {
 	describe('Get container revisions', () => {
-		const getRevisionList = () => [
-			{ _id: ServiceHelper.generateUUIDString(), author: 'user1', timestamp: new Date(), rFile: [`${ServiceHelper.generateUUIDString()}_${ServiceHelper.generateUUIDString()}_ifc`] },
-			{ _id: ServiceHelper.generateUUIDString(), author: 'user1', timestamp: new Date(), rFile: [`${ServiceHelper.generateUUIDString()}_${ServiceHelper.generateUUIDString()}_obj`] },
-			{ _id: ServiceHelper.generateUUIDString(), author: 'user1', timestamp: new Date() },
-			{ _id: ServiceHelper.generateUUIDString(), author: 'user1', timestamp: new Date(), void: true },
+		const revisions = [
+			{ _id: ServiceHelper.generateUUIDString(),
+				author: ServiceHelper.generateRandomString(),
+				timestamp: new Date(),
+				rFile: [`${ServiceHelper.generateUUIDString()}_${ServiceHelper.generateUUIDString()}_ifc`] },
+			{ _id: ServiceHelper.generateUUIDString(),
+				author: ServiceHelper.generateRandomString(),
+				timestamp: new Date(),
+				rFile: [`${ServiceHelper.generateUUIDString()}_${ServiceHelper.generateUUIDString()}_obj`] },
+			{ _id: ServiceHelper.generateUUIDString(),
+				author: ServiceHelper.generateRandomString(),
+				timestamp: new Date() },
+			{ _id: ServiceHelper.generateUUIDString(),
+				author: ServiceHelper.generateRandomString(),
+				timestamp: new Date(),
+				void: true },
 		];
 
-		const formatRevisions = (revisions) => {
-			revisions.map((r) => {
+		const formatRevisions = (revs) => {
+			revs.map((r) => {
```

Comment:
Same situation here as the source code version

But I think this also breaks the test since you're modifying the object?

maybe it's better to keep it as a map and return a copy of the object using the spread operator.